1-exercises/H-array-methods-2/exercise.js

```diff
@@ -15,8 +15,8 @@ let everyone = [
   "Swathi",
 ];
 
-let firstFive; // complete this statement
-let lastFive; // complete this statement
+let firstFive=everyone.slice(0,5) // complete this statement
+let lastFive=everyone.slice(2) // complete this statement
```

Comment:
This gives us the correct answer back in this use case, however, if the array length was greater, the logic here unfortunately breaks.  I would consider rewriting this so we are explicitly asking for the last 5 items regardless of however long the array may be.